etc/config/toit.defaults.properties

```diff
@@ -0,0 +1,6 @@
+compilers=/usr/bin/toit
```

Comment:
Is this meant to be also live in the site? if so, it's missing some things:
- A PR in the sister repo https://github.com/compiler-explorer/infra that installs the compiler to the required places (There are some example PRs you can check out)
- And a `toit.amazon.properties` files to configure the compilers for the live site

Also, you should add the language to `.github/labeler.yml`